gridlayout.h

```diff
@@ -196,6 +198,10 @@ class Grid
                         }
                         sprite_set_x(_grid[index].button->btn(), x);
                         sprite_set_y(_grid[index].button->btn(), y);
+                        // Update Button position of button
+                        this->_grid[index].button->set_x(sprite_x(this->_grid[index].button->btn()));
+                        this->_grid[index].button->set_y(sprite_y(this->_grid[index].button->btn()));
```

Comment:
Unsure why the position of the sprite isn't just acquired from use of the [sprite_position](https://splashkit.io/api/sprites/#sprite-position) function, to ensure consistency is maintained across the code base.

Reply:
_set_x/set_y_ was used to maintain consistency with existing _sprite_set_x/sprite_set_y_ used above.